content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -5,8 +5,181 @@ description: >
+      #          - csi-isilon.dellemc.com
+
+      # specify additional mount options for when a Persistent Volume is being mounted on a node.
+      # To mount volume with NFSv4, specify mount option vers=4. Make sure NFSv4 is enabled on the Isilon Cluster
+      #mountOptions: ["<mountOption1>", "<mountOption2>", ..., "<mountOptionN>"]
+    ```
+
+    Save the file and create it by using `kubectl create -f <input_file.yaml>`.
+
+### Install CSM Authorization Proxy Server
+
+1. Follow all the [prerequisites](#prerequisite).
+
+2. Create a CR (Custom Resource) for Authorization using the sample file provided [here](https://github.com/dell/csm-operator/blob/main/samples/authorization/csm_authorization_proxy_server.yaml). This file can be modified to use custom parameters if needed.
+
+3. Users should configure the parameters in the CR. This table lists the primary configurable parameters of the Authorization Proxy Server and their default values:
+
+   | Parameter | Description | Required | Default |
+   | --------- | ----------- | -------- |-------- |
+   | **authorization** | This section configures the CSM-Authorization components. | - | - |
+   | PROXY_HOST | The hostname to configure the self-signed certificate (if applicable), and the proxy, tenant, role, and storage service Ingresses. | Yes | csm-authorization.com |
+   | AUTHORIZATION_LOG_LEVEL | CSM Authorization log level. Allowed values: “error”, “warn”/“warning”, “info”, “debug”. | Yes | debug |
+   | AUTHORIZATION_ZIPKIN_COLLECTORURI | The URI of the Zipkin instance to export traces. | No | - |
+   | AUTHORIZATION_ZIPKIN_PROBABILITY | The ratio of traces to export. | No | - |
+   | PROXY_INGRESS_CLASSNAME | The ingressClassName of the proxy-service Ingress. | Yes | nginx |
+   | PROXY_INGRESS_HOSTS | Additional host rules to be applied to the proxy-service Ingress.  | No | authorization-ingress-nginx-controller.authorization.svc.cluster.local |
+   | TENANT_INGRESS_CLASSNAME | The ingressClassName of the tenant-service Ingress. | Yes | nginx |
+   | ROLE_INGRESS_CLASSNAME | The ingressClassName of the role-service Ingress. | Yes | nginx |
+   | STORAGE_INGRESS_CLASSNAME | The ingressClassName of the storage-service Ingress. | Yes | nginx |
+   | REDIS_STORAGE_CLASS | The storage class for Redis to use for persistence. If not supplied, the default storage class is used. | Yes | - |
+   | **ingress-nginx** | This section configures the enablement of the NGINX Ingress Controller. | - | - |
+   | enabled | Enable/Disable deployment of the NGINX Ingress Controller. Set to false if you already have an Ingress Controller installed. | No | true |
+   | **cert-manager** | This section configures the enablement of cert-manager. | - | - |
+   | enabled | Enable/Disable deployment of cert-manager. Set to false if you already have cert-manager installed. | No | true |
+
+4. Execute this command to create Authorization CR:
+
+    ```kubectl create -f <input_sample_file.yaml>``` .
```

Comment:
Can we just call this csm_authorization_proxy_server.yaml instead of input_sample_file.yaml to avoid confusion.

Reply:
fixed.